docs/types/sequences-sets/lists.rst

```diff
@@ -30,7 +30,7 @@ Indizes
 Elemente können aus einer Python-Liste extrahiert werden, indem eine Notation
 verwendet wird, die der Array-Indizierung in C ähnelt und mit ``0`` beginnt; die
 Frage nach Element ``0`` gibt das erste Element der Liste zurück, die Frage nach
-Element ``1`` gibt das zweite Element zurück :abbr:`u.s.w. (und so weiter)`.
+Element ``1`` gibt das zweite Element zurück :abbr:`usw. (und so weiter)`.
```

Comment:
Vermutlich besser wäre, den Punkt in der Abkürzung wegzulassen damit nicht zwei Punkte aufeinander folgen.